### PR TITLE
[2.0] update 2.0.0 benchmarks

### PR DESCRIPTION
**Pull request by scv119**
## Why are these changes needed?

Update the benchmark results for 2.0.0 releases. 

generated by
```
(doc) chenshen@Chens-MacBook-Pro-16-inch-2019:~/Documents/source/ray/release/release_logs(master)$ python fetch_release_logs.py 2.0.0rc1
Missing BUILDKITE_TOKEN, retrieving from AWS secrets store
Fetched microbenchmark.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched stress_tests/stress_test_many_tasks.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched stress_tests/stress_test_dead_actors.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched stress_tests/stress_test_placement_group.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched scalability/single_node.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched scalability/object_store.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched benchmarks/many_actors.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched benchmarks/many_tasks.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched benchmarks/many_pgs.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Fetched benchmarks/many_nodes.json for commit 72874f2e371ab66d0e4cf38eb417f70f3356e489
Writing 2.0.0rc1/microbenchmark.json
Writing 2.0.0rc1/stress_tests/stress_test_many_tasks.json
Writing 2.0.0rc1/stress_tests/stress_test_dead_actors.json
Writing 2.0.0rc1/stress_tests/stress_test_placement_group.json
Writing 2.0.0rc1/scalability/single_node.json
Writing 2.0.0rc1/scalability/object_store.json
Writing 2.0.0rc1/benchmarks/many_actors.json
Writing 2.0.0rc1/benchmarks/many_tasks.json
Writing 2.0.0rc1/benchmarks/many_pgs.json
Writing 2.0.0rc1/benchmarks/many_nodes.json
```

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/release_logs/2.0.0/scalability/single_node.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        {
+            "perf_metric_name": "10000_get_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 29.429414217000044
+        },
+        {
+            "perf_metric_name": "1000000_queued_time",
+            "perf_metric_type": "LATENCY",
+            "perf_metric_value": 170.71477117199998
```
Comment (scv119) [inline release/release_logs/2.0.0/scalability/single_node.json:29]: this was 158.97089586599998

File: release/release_logs/2.0.0/stress_tests/stress_test_placement_group.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "avg_pg_create_time_ms": 0.6444583648660457,
+    "avg_pg_remove_time_ms": 0.6946435885894873,
```
Comment (scv119) [inline release/release_logs/2.0.0/stress_tests/stress_test_placement_group.json:3]: wow what happened here? it was 3.11 @rkooo567

**Comment by scv119** [comment]
@jjyao @rkooo567 take another look?

File: release/release_logs/2.0.0/benchmarks/many_actors.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "_peak_memory": 5.2,
+    "_peak_process_memory": "PID\tMEM\tCOMMAND\n191\t1.53GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n1807\t0.93GiB\tpython distributed/test_many_actors.py\n216\t0.26GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dash\n285\t0.11GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n80\t0.09GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n1847\t0.06GiB\tray::MemoryMonitorActor.run()\n209\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.65.11:9031 --host=0.0.0.0\n254\t0.05GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/log_m\n79\t0.05GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=2f874416\n247\t0.02GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/raylet/raylet --raylet_socket_name=",
```
Comment (matthewdeng) [inline release/release_logs/2.0.0/benchmarks/many_actors.json:3]: This looks wrong?

**Review by matthewdeng — COMMENTED** [review]
Maybe we can hold off on committing this, this is probably good to sanity check rc1 but we should wait until we have the final 2.0.0 wheel to record?

**Review by jjyao — APPROVED** [review]
LGTM

**Comment by scv119** [comment]
sgtm

**Pull request merged**